src/main/java/org/spongepowered/common/data/processor/data/item/ColoredDataProcessor.java

```diff
@@ -38,14 +40,19 @@
 import org.spongepowered.common.data.util.NbtDataUtil;
 import org.spongepowered.common.data.value.immutable.ImmutableSpongeValue;
 import org.spongepowered.common.data.value.mutable.SpongeValue;
-import org.spongepowered.common.util.ColorUtil;
 
 import java.util.Optional;
 
 public class ColoredDataProcessor extends AbstractItemSingleDataProcessor<Color, Value<Color>, ColoredData, ImmutableColoredData> {
 
     public ColoredDataProcessor() {
-        super(ColorUtil::hasColor, Keys.COLOR);
+        super(ColoredDataProcessor::supportsColor, Keys.COLOR);
+    }
+
+    private static boolean supportsColor(ItemStack stack) {
```

Comment:
Any chance this can be overridden in Forge for compatibility with other mods?

Reply:
Unsure, as my experience with both Forge and the Data API is lacking. This method itself was just copied from `ColorUtil` so I really don't know the specifics of it and what would need to be done to support this.